src/pages/SearchPage/index.js

```diff
@@ -0,0 +1,183 @@
+import PropTypes from 'prop-types';
+import React, {useEffect, useMemo, useState} from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import {usePersonalDetails} from '@components/OnyxProvider';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import useDebouncedState from '@hooks/useDebouncedState';
+import useLocalize from '@hooks/useLocalize';
+import useNetwork from '@hooks/useNetwork';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import * as OptionsListUtils from '@libs/OptionsListUtils';
+import Performance from '@libs/Performance';
+import * as ReportUtils from '@libs/ReportUtils';
+import reportPropTypes from '@pages/reportPropTypes';
+import * as Report from '@userActions/Report';
+import Timing from '@userActions/Timing';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import SearchPageFooter from './SearchPageFooter';
+
+const propTypes = {
+    /* Onyx Props */
+
+    /** Beta features list */
+    betas: PropTypes.arrayOf(PropTypes.string),
+
+    /** All reports shared with the user */
+    reports: PropTypes.objectOf(reportPropTypes),
+};
+
+const defaultProps = {
+    betas: [],
+    reports: {},
+};
+
+const setPerformanceTimersEnd = () => {
+    Timing.end(CONST.TIMING.SEARCH_RENDER);
+    Performance.markEnd(CONST.TIMING.SEARCH_RENDER);
+};
+
+const SearchPageFooterInstance = <SearchPageFooter />;
+
+function SearchPage({betas, reports}) {
```

Comment:
@lukemorawski the skeleton on search page is not implemented

https://github.com/Expensify/App/assets/29673073/b0c877b8-69a7-4b1d-ae47-0842b6fc2357

Staging: 

<img width="289" alt="image" src="https://github.com/Expensify/App/assets/29673073/2658c079-770f-4782-870b-9e195d271dd1">

Reply:
fixed that